mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -26,6 +26,19 @@
         
         expect(sourcepage.errors[:source]).to include("Please at least add one type of source")
     end
+
+    it 'creates website card with actions' do
+      url = 'http://www.google.com/?q=wikirateissocoolandawesomeyouknow'
+      Card::Env.params[:sourcebox] = 'true'
+      sourcepage = Card.create! type_id: Card::SourceID, subcards: {
+        '+Link' => { content: url },
+        '+File' => { type_id: Card::FileID },
+        '+Text' => { type_id: Card::BasicID, content: '' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.